core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -333,7 +337,8 @@ public HttpResponse doConnectionStatus(StaplerRequest request) {
                 }
                 return HttpResponses.okJSON(checkJob.connectionStates);
             } else {
-                return HttpResponses.errorJSON(String.format("Unknown site '%s'.", siteId));
+                return HttpResponses.errorJSON(String.format("Cannot check connection status of the update site with ID='%s'"
+                        + ". This update center cannot be resolved", siteId));
```

Comment:
Right. I messed up the original formatting a bit during the warning text integration. No other usages on the other hand